src/events.rs

```diff
@@ -106,7 +109,7 @@ impl Events {
             match val {
                 Some(v) => {
                     body.push_str(&v);
-                    body.push_str("%0A");
+                    body.push_str("\n");
```

Comment:
I assume it's ok to have a newline at the end of the final request line, as opposed to only being in between two requests.